Makefile

```diff
@@ -13,7 +13,7 @@ export ARCH_TYPE   = $(if $(patsubst x86_64,,$(ARCH)),$(ARCH),amd64)
 export BUILD_DATE  = $(shell date +%m/%d@%H:%M:%S)
 export VCS_REF     = $(if $(shell git status --porcelain),$(GIT_COMMIT)-$(BUILD_DATE),$(GIT_COMMIT))
 
-export CGO_ENABLED  = 1
```

Comment:
In the process of upgrade I met the same issue of: https://redhat-internal.slack.com/archives/C02A1PX4B9S/p1686886726859979

```
/lib64/libc.so.6: version `GLIBC_2.34' not found
```

To solve this I updated the CGO_ENABLED by default to 0.

Reply:
But do you remember why in the first place we change it to `1` by this PR https://github.com/stolostron/managedcluster-import-controller/pull/235 ? @elgnay